docs/guide/contributing.rst

```diff
@@ -4,18 +4,16 @@
 Contributing
 ************
 
-We welcome contributions -- if you are a localiser then this is your document.
-Please think of this as a living document.  If you are using part of it and
-find that it is out of date then please update it so that others don't have to
-go through your pain.
+We welcome contributions; if you are a localiser then this is your document.
+Please think of it as a living document. If you are using any part of it and
+find that it is out of date, then please update it so that others do not have to go through your pain.
```

Comment:
RST and MD files are manually trimmed to be roughly a fixed width, so that the lines are neat and readable.

Could you add a line-feed after `have to`, so that `go through...` is on the next line.